src/crimson/osd/main.cc

```diff
@@ -186,8 +186,13 @@ int main(int argc, const char* argv[])
           const auto nonce = crimson::osd::get_nonce();
           crimson::net::MessengerRef cluster_msgr, client_msgr;
           crimson::net::MessengerRef hb_front_msgr, hb_back_msgr;
+          for (auto [msgr, name] : {make_pair(std::ref(client_msgr), "client"s)}) {
```

Comment:
Probably simpler to skip the loop since there's only one element?

Reply:
yes, cluster msgr need multicore, you can enable it in this PR together, I have tested it many times, no problem.

Reply:
Thanks for noting! I'll push it in a follow-up PR.